armi/settings/fwSettings/globalSettings.py

```diff
@@ -109,7 +109,7 @@
 CONF_BLOCK_AUTO_GRID = "autoGenerateBlockGrids"
 CONF_INPUT_HEIGHTS_HOT = "inputHeightsConsideredHot"
 CONF_CYCLES = "cycles"
-CONF_USER_PLUGINS = "userPlugins"
+CONF_USER_PLUGINS = "UserPlugins"
```

Comment:
other settings are lower camel case, why is user plugin different?

Reply:
Fixed!